src/sass/components/_contacts.scss

```diff
@@ -0,0 +1,136 @@
+.contacts{
+    background-color: $pageBg;
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items:center;
+    padding:86px 50px 113px;
+    
+    
+
+
+&__head{
+    display:none;
+}
+    
+&__box{ 
+    display:flex;
+    flex-direction: column;
+    gap:20px;
+    max-width:480px;
+    width:100%;
+    justify-content: center;
+   
+
+}
+    
+
+
+&__container{
+    background-color:$white;
+    box-shadow:0px 8px 30px $contactsBoxShadow;
+    border-radius:24px;
+    padding:43px 20px;
+    
+}
+  
+
+
+&__title{
+@include font-mixin(inherit, 500, 12px, 1.86, 0.04em, $white);
+border-radius:6px;
+text-align: center;
+padding:2px 25px;
+
+
+    
+}
+&__title--cafe{
```

Comment:
tutaj możesz i w sumie powinieneś zagnieździć to w modyfikowanym elemencie, czyli robisz "&--cafe"